components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -44,4 +44,94 @@
+                text="Gitpod: Documentation"
+                icon="AllIcons.General.ReaderMode">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.ReportIssueAction"
+                class="io.gitpod.jetbrains.remote.actions.ReportIssueAction"
+                text="Gitpod: Report Issue"
+                icon="AllIcons.General.ShowWarning">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.FollowUsOnTwitterAction"
+                class="io.gitpod.jetbrains.remote.actions.FollowUsOnTwitterAction"
+                text="Gitpod: Follow Us On Twitter"
+                icon="AllIcons.Gutter.ExtAnnotation">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.CommunityChatAction"
+                class="io.gitpod.jetbrains.remote.actions.CommunityChatAction"
+                text="Gitpod: Open Community Chat"
+                icon="AllIcons.General.Balloon">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.UpgradeSubscriptionAction"
+                class="io.gitpod.jetbrains.remote.actions.UpgradeSubscriptionAction"
+                text="Gitpod: Upgrade Subscription"
+                icon="AllIcons.Actions.PreviousOccurence">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <!-- The following actions have their class and icons defined, but still need to be implemented.
+        <action id="io.gitpod.jetbrains.remote.actions.StopWorkspaceAction"
+                class="io.gitpod.jetbrains.remote.actions.StopWorkspaceAction"
+                text="Gitpod: Stop Workspace"
+                icon="AllIcons.Actions.Pause">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.ExtendWorkspaceTimeoutAction"
+                class="io.gitpod.jetbrains.remote.actions.ExtendWorkspaceTimeoutAction"
+                text="Gitpod: Extend Workspace Timeout"
+                icon="AllIcons.Vcs.History">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.ShareWorkspaceSnapshotAction"
+                class="io.gitpod.jetbrains.remote.actions.ShareWorkspaceSnapshotAction"
+                text="Gitpod: Share Workspace Snapshot"
+                icon="AllIcons.Nodes.UpFolder">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.ShareRunningWorkspaceAction"
```

Comment:
this one can be removed, live sharing like that does not work with JB, only owner can connect